### ISSUE TITLE
Slow import

### ISSUE THREAD

**Issue by gnumoksha** [phpmyadmin/docker#163]
Hi. phpMyAdmin takes ~2 minutes to import a SQL dump of 1.4MiB.

```
Version information: 4.8.0.1 (up to date)
```

```
$ docker version
Client:
 Version:      18.03.1-ce
 API version:  1.37
 Go version:   go1.9.5
 Git commit:   9ee9f40
 Built:        Thu Apr 26 07:17:31 2018
 OS/Arch:      linux/amd64
 Experimental: false
 Orchestrator: swarm

Server:
 Engine:
  Version:      18.03.1-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.5
  Git commit:   9ee9f40
  Built:        Thu Apr 26 07:15:42 2018
  OS/Arch:      linux/amd64
  Experimental: false
```

docker-compose.yml
```yaml
version: '3.5'

services:
  mysql:
    image: mysql:5.7
    environment:
      - MYSQL_DATABASE=my_litle_database
      - MYSQL_ROOT_PASSWORD=my_litle_password
    volumes:
      - database:/var/lib/mysql
    ports:
      - 3306:3306
    deploy:
      placement:
        constraints: [node.role == manager]

  phpmyadmin:
    image: phpmyadmin/phpmyadmin
    container_name: phpmyadmin
    environment:
      - PMA_ARBITRARY=1
    links:
      - mysql:db
    ports:
      - 8181:80
    volumes:
      - /phpmyadmin-sessions

```

**Comment by ibennetch** [comment]
This is probably the same as https://github.com/phpmyadmin/phpmyadmin/issues/14123.

Basically, it's in the issue tracker as a problem, so it's a known issue at this point and there has been some headway made towards finding the root cause, but a fix hasn't yet been implemented.

**Comment by gnumoksha** [comment]
Thank you for the quick reply. Is there any test I can do or information I could provide?

**Comment by williamdes** [comment]
@gnumoksha I think you should try to use the latest 4.9 version available using `4.9` tag on docker hub

**Comment by gnumoksha** [comment]
Ok, thank you. I will close this since I'm not affected by the issue anymore.
